/vslam:
  ros__parameters:
    # ORB params
    orb_scale_pyramid_levels: 8
    orb_pyramid_scale_factor: 1.2
    orb_num_features: 1000
    orb_initial_FAST_threshold: 20
    orb_min_FAST_threshold: 7

    # Camera params
    camera_fx: 517.306408
    camera_fy: 516.469215
    camera_cx: 318.643040
    camera_cy: 255.313989
    camera_k1: 0.262383
    camera_k2: -0.953104
    camera_p1: -0.005358
    camera_p2: 0.002628
    camera_k3: 1.163314
    camera_width: 640
    camera_height: 480
    camera_fps: 30
    camera_depth_threshold: 40.0
    camera_depth_baseline_mm: 80.0
    camera_depth_map_factor: 5000.0
